src/data_base.py

```diff
@@ -1,55 +1,71 @@
-import sqlite3
 from typing import List
+from urllib import parse
 
+import psycopg2
+from psycopg2 import sql
+
+from src.environment_variables import get_env
 from src.photo import Photo
 
 
 class DataBase:
-    def __init__(self, file_path="src.test.db"):
-        self.file_path = file_path
-        self.db = sqlite3.connect(self.file_path)
-        self.db.execute('''CREATE TABLE IF NOT EXISTS photos_list 
-                        (id VARCHAR, 
-                        posted BOOL, 
-                        UNIQUE(id) )''')
+    def __init__(self, table_name: str = "photos_list"):
+        self.table_name = table_name
+        parse.uses_netloc.append("postgres")
+        url = parse.urlparse(get_env("HEROKU_DATABASE_URL"))
+        self.connection = psycopg2.connect(
+            database=url.path[1:],
+            user=url.username,
+            password=url.password,
+            host=url.hostname,
+            port=url.port
+        )
+        self.cursor = self.connection.cursor()
+        with self.connection as connection:
+            connection.cursor().execute(sql.SQL('''CREATE TABLE IF NOT EXISTS {}
+                                                    (id VARCHAR,
+                                                    posted BOOL,
+                                                    UNIQUE(id) )''').format(sql.Identifier(self.table_name)))
 
     def __enter__(self):
         return self
 
-    def __exit__(self):
-        self.db.close()
-
-    def add_photo(self, id_photo):
-        self.db.execute("INSERT OR IGNORE INTO photos_list(id, posted) VALUES(?,0)", (id_photo,))
-        self.db.commit()
+    def __exit__(self, *args, **kwargs):
+        self.cursor.close()
```

Comment:
Move these two commands to a new .close() method and call it here

Reply:
also fixed db fixture in test_data_base